climada/hazard/storm_europe.py

```diff
@@ -199,6 +206,209 @@ def _read_one_nc(self, file_name, centroids):
         ncdf.close()
         return new_haz
 
+
+    def read_cosmoe_file(self, fp_file, run_date, event_date=None,
```

Comment:
the function doesn't just read a cosmo-e file, right? it reads a file AND sets a hazard from this file.
I'd perhaps suggest that you either refactor the actual reading part (until line 259), or rename the function into `set_storm_from_cosmoe` (or sth of the sort)

Reply:
good point - we could also think about renaming the existing `read_footprints` and `_read_one_nc` methods so it's clear they're for WISC. (but how much would that break backward compatibility?)

Reply:
I would prefer to not rename the read_footprints function for backwards compatibility. That is why I would name the new functions consistently (at least consistently within this class). Is that ok?

Reply:
yeah, let's not mess with backward compatibility then.
since the method will be called via `StormEurope.read_cosmoe_file()` i dont think the name is too unclear, but something like evelyn's suggestion would make it super obvious what it's doing